rules/rules.go

```diff
@@ -18,6 +19,12 @@ type Rule struct {
 	Regexp *Regexp `json:"regexp"`
 }
 
+// MatchHidden matches paths with a basename
+// that begins with a dot.
+func MatchHidden(path string) bool {
+	return filepath.Base(path)[0] == '.'
```

Comment:
It will cause panic on empty string. 
```suggestion
	return strings.HasPrefix(filepath.Base(path, "."))
```